docs/getting-started/db.md

```diff
@@ -4,15 +4,18 @@ title: Preparing the Database
 
 # Preparing the database
 
-We need to create our database, load the database schema, and load seed
-data. This can be accomplished with:
+The next step is to create and prepare the database. Thankfully, dev.to is a
+Rails application, so this is relatively easy.
+
+We can use Rails to create our database, load the schema, and add some seed
+data:
 
 ```shell
 rails db:setup
 ```
 
 `db:setup` actually runs the following rake commands in order so
-you could alternatively run each of these to produce the same result.
+alternatively, you could run each of these to produce the same result.
```

Comment:
I'd replace the period with a colon since we have "following commands"

Reply:
Sounds good to me :)